spec/fixtures/pam_password_auth_ac.defaults.el6

```diff
@@ -0,0 +1,24 @@
+# This file is being maintained by Puppet.
```

Comment:
it is read in line 314 where check can be 'defaults' or 'vas'
`fixture = File.read(fixtures("#{filename}.#{check}.#{k}"))`
`it { should contain_file(filename).with_content(fixture) }``